spec/lib/mcb/commands/providers/accredited_course_sites_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "mcb_helper"
+
+describe "mcb providers accredited_course_sites" do
+  def execute(arguments: [], input: [])
+    with_stubbed_stdout(stdin: input.join("\n")) do
+      $mcb.run(["providers", "accredited_course_sites", *arguments])
+    end
+  end
+
+  let(:provider) { create :provider, :accredited_body }
+  let!(:findable_course) do
+    create :course, name: "findable-course",
+           accrediting_provider: provider,
+           site_statuses: [build(:site_status, :findable)]
+  end
+
+  it "writes course information to terminal" do
+    output = execute(arguments: [provider.provider_code])[:stdout]
+    expect(output).to include("provider_code | provider_name") # headers
```

Comment:
How about using `have_text_table_row`? Look for examples in existing code.

Reply:
we can then validate all the column names without worrying about whitespace